README.rst

```diff
@@ -32,6 +32,16 @@ Installation
 See doc/install.txt in the source code archive.
 Python 2.7.2 or later is needed.
 
+
+Install using Docker
+---------------------
+You can try linkchecker in an isolated enviroment without changing any local files or have to install all the dependencies via a Docker Container:
+
+``docker pull ellerbrock/linkchecker``
+
+https://hub.docker.com/r/ellerbrock/link-checker/
```

Comment:
can you be more explicit here? what is this link? how is it related to the project?

maybe mention that this is not an official build? otherwise why "ellerbrock" instead of "linkchecker"?

how often are those images rebuilt? what version?

so many questions... :)

i'm not very familiar with docker, so maybe that makes no sense at all... :)